SECURITY.md

```diff
@@ -56,4 +56,13 @@ dzasNSG79KqWv6JkzwvyNeviaEc74xHe3YT4iqVpX7aZtmdLS+oAR5f9vq+ksL5m
 QQM22M3gnZQsefEYucpqHsnQx7s=
 =mWRd
 -----END PGP PUBLIC KEY BLOCK-----
-```
\ No newline at end of file
+```
+
+# Random number generation
+
+Depending on the system a different source of a random number generator is
+used. Full explanation can be found in [Go
+documentation](https://golang.org/pkg/crypto/rand/#pkg-variables).
+
+On Windows [CryptoGenRandom](https://en.wikipedia.org/wiki/CryptGenRandom) is
```

Comment:
I think we'll need to elaborate on which parts of our system (Ed25519 key gen) are affected by this, otherwise seems kind of vague.